microsoft-365/admin/setup/customize-your-organization-theme.md

```diff
@@ -59,23 +59,23 @@ Learn how to customize your theme in the Microsoft 365 admin center. As the admi
     
 4. Select **Save changes**.
     
-You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Office 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/en-us/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/en-us/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). See the following graphic for an example of where you an customize theme changes from the admin center.
+You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Microsoft 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/en-us/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/en-us/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). For an example of where you an customize theme changes from the admin center, see the following image.
```

Comment:
```suggestion
You'll see your new theme on the admin center right away and after a short delay, you'll see it throughout Microsoft 365, including pages in Outlook, SharePoint, [SharePoint mobile app for iOS](https://support.office.com/article/SharePoint-mobile-app-for-iOS-339402ce-16bb-4c97-9475-0c5375ccef7a), and [SharePoint mobile app for Android](https://support.office.com/article/SharePoint-mobile-app-for-Android-d875654b-fb0a-4dbe-a17a-a676cf936284). For an example of where you can customize theme changes from the admin center, see the following image.
```

Reply:
Please change **an** in the last sentence to **can**. Also remove locales from links.